docs/source/create/avatars/avatar-standards.md

```diff
@@ -173,8 +173,14 @@ When you wear different avatars, you'll notice that each avatar has a different
 
 Depending on the avatar's design, the collision hulls can be very large or small. This occurs because High Fidelity analyzes the shape of the avatar's torso (from hips to head) and tries to find the best shape that encloses the mesh. For example, if your avatar has large hips or perhaps a fully extended tail, High Fidelity thinks that the tip of the tail is part of your hips, and makes a very large collision hull.  To reduce the size of the collision hull, you can add skeleton joints to your avatar's tail. 
 
+
+### Avatar Thumbnails
+
+The Avatar app will look for a thumbnail with the same URL as your avatar but a .jpg extension.
+For example, if your avatar is hosted on `http://vircadia.dev/avatar.fbx`, then it will look for a thumbnail at `http://vircadia.dev/avatar.jpg`.
```

Comment:
```suggestion
For example, if your avatar is hosted at `https://vircadia.dev/avatar.fbx`, then it will look for a thumbnail at `https://vircadia.dev/avatar.jpg`.
```